packages/ui/src/components/Accordion/Accordion.css.ts

```diff
@@ -0,0 +1,33 @@
+import { tokens } from '@repo/theme';
+import { style } from '@vanilla-extract/css';
+
+export const accordionRoot = style({
+  width: '100%',
```

Comment:
```suggestion
  width: 'fit-content',
```
P2: 지금 100%라서 화면 너비 전체 clickable한 상태입니다! 제가 만든 Checkbox도 같은 상황이라 추후 수정하려구요!

Reply:
앗 네넵 수정하겠습니다!